src/pretix/control/forms/item.py

```diff
@@ -156,6 +188,20 @@ def save(self, *args, **kwargs):
 
         instance = super().save(*args, **kwargs)
 
+        if not self.cleaned_data.get('quota_option') == '0':
+            if self.cleaned_data.get('quota_option') == '1' and self.cleaned_data.get('quota_add_existing') is not None:
+                quota_name = self.cleaned_data.get('quota_add_existing').name
+                quota = self.instance.event.quotas.filter(name=quota_name).get()
+                quota.items.add(self.instance)
```

Comment:
Why don't you just use the quota object?
```
quota = self.cleaned_data.get('quota_add_existing')
quota.items.add(self.instance)
```
Getting it again by name is not only useless, but also dangerous (there could be two with the same name for example)